templates/base.html

```diff
@@ -70,5 +78,7 @@
         {% endblock %}
         <script src="{% static 'scripts/global_search.js' %}"></script>
         <script src="{% static 'scripts/hamburger_menu.js' %}"></script>
+        <script src="{% static 'cookie-consent/ds-cookie-consent.js' %}"></script>
+
```

Comment:
This code is cookie related, so might need to be wrapped in a conditional.

Reply:
corrected, this is now included to show based on the value FEATURE_COOKIE_BANNER_ENABLED